packages/data-point-codemods/transforms/__testfixtures__/change-path-reducer-accessing-root-path-double-quote.input.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+/* eslint-disable prettier */
+
+// prettier-ignore
+const entities = {
+  'transform:a': `$.`,
+  'transform:b': `$. | foo | $.`,
+  'transform:c': `foo | bar | baz`,
+  'transform:d': "$.",
+  'transform:e': "$. | foo",
+  'transform:f': "$. | $. | $.",
+  'transform:g': "foo | bar | baz",
+  nested: {
+    object: {
+      a: "$. | $."
+    },
+    array: ["$.", "'$.", "$."]
+  }
+}
+
+dataPoint.transform('foo | $. | baz | $.')
```

Comment:
@raingerber Covering a few more scenarios.

Reply:
Wow. I overlooked this bug, pushed it, and then commented on it, and I _still_ didn’t notice until the CI failed. 🤦‍♂️